app/controllers/cats_controller.rb

```diff
@@ -1,30 +1,33 @@
 class CatsController < ApplicationController
 
-#         def index
-#             cats = Cat.all
-#             render json: cats
-#           end
+    def index
+        cats = Cat.all
+        render json: cats
+    end
       
-#           def create
-#             # Create a new cat
-#             cat = Cat.create(cat_params)
-#             render json: cat
-#           end
-        
-#           # Handle strong parameters, so we are secure
-#           private
-#           def cat_params
-#             params.require(:cat).permit(:name, :age, :enjoys, :image)
-#           end
-      
-#         def update
-#         end
-      
-#         def destroy
-#         end
-      
-#       end
+    def create
+    # Create a new cat
+    cat = Cat.create(cat_params)
+    render json: cat
+end
 
+    def update
+    cat = Cat.find(params[:id])
+    cat.update(cat_params)
+    if cat.valid?
+      render json: cat
+    else
+      render json: cat.errors
+    end
+  end
+  
+def destroy
```

Comment:
If you don't have specs for update and destroy, you should remove the controller methods.